impeller/entity/contents/content_context.cc

```diff
@@ -587,43 +587,6 @@ void ContentContext::InitializeCommonlyUsedShadersIfNeeded() const {
     return;
   }
 
-  // Initialize commonly used shaders that aren't defaults. These settings were
-  // chosen based on the knowledge that we mix and match triangle and
-  // triangle-strip geometry, and also have fairly agressive srcOver to src
-  // blend mode conversions.
```

Comment:
There is a comment above that references this if I remember correctly, we should remove that too.

Reply:
removed

Reply:
I think you forgot to push it.  It's this one:
```
  // These pipelines are created first since they are immediately used by
  // InitializeCommonlyUsedShadersIfNeeded. Their order matches the order in
  // InitializeCommonlyUsedShadersIfNeeded.
```

Reply:
oh that one!

Reply:
Done